integration-tests/README.md

```diff
@@ -31,9 +31,23 @@ import { ALITH_ADDRESS } from "@moonwall/util";
 
 ### Usage Examples (non-exhaustive)
 
+You should have installed `python` for using `sqlite3` and then used `pnpm rebuild && pnpm rebuild sqlite3`.
+
+Useful for integration testing:
+
+- `./integration-tests/scripts/deploy_zombienet.sh` - Deploy a local relay-parachain network for zombienet.
```

Comment:
When running the script `deploy_zombienet.sh` there is no need to run `cargo build --release --features parachain` since this is already done inside the script (but debug version of it). 

It seems like that you are not able to properly download the zombienet macos executable... mhh

This command seems to not work for you `curl -L "https://github.com/paritytech/zombienet/releases/download/v1.3.89/zombienet-macos" -o "./integration-tests/tmp/zombienet"` I wonder why..

Reply:
I wonder why you get `Please make sure you executed 'cargo build --release --features parachain'.` as output, because I can't see this on my current version of the code. I think I had it in the past.

Reply:
6d79feb96e087404c3a77b0e171a4fb5a772876c

Reply:
6d79feb96e087404c3a77b0e171a4fb5a772876c

Reply:
Adding `mkdir -p ./integration-tests/tmp` helped.

Reply:
5bd843ae82db3f1f9b991c89bc1158d90b9c84e8

Reply:
Now I'm getting this:

```
Error:  	 Error: Command failed with exit code 127: bash -c ./integration-tests/tmp/polkadot build-spec --chain rococo-dev --disable-default-bootnode > /var/folders/61/mf6sr7b900jcmtd_dfw_z_rr0000gn/T/zombie-452b14969d7c7148d45a8ae5780229b8_-23740-fICuIb4GeQaH/cfg/rococo-dev-plain.json
bash: ./integration-tests/tmp/polkadot: No such file or directory
```

Seems like this test expected a polkadot binary as well.

Reply:
bump

Reply:
On which machine did you execute the command?

Reply:
This is all on macos

Reply:
This is related to https://github.com/zeitgeistpm/zeitgeist/pull/1194#discussion_r1461823079. I just failed to document running the download-polkadot script beforehand.

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/1194/commits/6e574ff3536bab823e47f4bebbb124232de521b2